packages/atlas-service/src/atlas-service.ts

```diff
@@ -35,19 +38,25 @@ export class AtlasService {
   cloudEndpoint(path?: string): string {
     return encodeURI(`${this.config.cloudBaseUrl}${path ? `/${path}` : ''}`);
   }
+  regionalizedCloudEndpoint(
+    _atlasMetadata: Pick<AtlasClusterMetadata, 'regionalBaseUrl'>,
+    path?: string
+  ): string {
+    // TODO: eventually should apply the regional url logic
```

Comment:
Is this part of another ticket/epic?

Reply:
I'll try to see if we have a project already, but this would also involve regionalizing the new backend and this is a non-goal currently and I'm not sure when exactly we're planning to get to it